[2022-12-12 21:09:10,625] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T20:37:00+00:00 [queued]>
[2022-12-12 21:09:11,000] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T20:37:00+00:00 [queued]>
[2022-12-12 21:09:11,018] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 21:09:11,029] {taskinstance.py:1357} INFO - Starting attempt 2 of 1
[2022-12-12 21:09:11,031] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 21:09:11,448] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-11-28 20:37:00+00:00
[2022-12-12 21:09:11,605] {standard_task_runner.py:52} INFO - Started process 213 to run task
[2022-12-12 21:09:11,781] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-11-28T20:37:00+00:00', '--job-id', '742', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpvzdh07nz', '--error-file', '/tmp/tmpmp8lz2mz']
[2022-12-12 21:09:11,795] {standard_task_runner.py:80} INFO - Job 742: Subtask get_last_id
[2022-12-12 21:09:13,383] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T20:37:00+00:00 [running]> on host cfea470654c7
[2022-12-12 21:09:15,890] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T20:37:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T20:37:00+00:00
[2022-12-12 21:09:16,218] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-12 21:09:16,904] {python.py:173} INFO - Done. Returned value was: None
[2022-12-12 21:09:17,424] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221128T203700, start_date=20221212T210910, end_date=20221212T210917
[2022-12-12 21:09:17,773] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-12 21:09:18,542] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
